console/console-init/ui/src/modules/connection-detail/components/ConnectionLinksToolbar/ConnectionLinksToolbar.tsx

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import {
+  DataToolbarItem,
+  DataToolbar,
+  DataToolbarContent,
+  SelectOptionObject,
+  DataToolbarChipGroup,
+  DataToolbarChip,
+  DataToolbarContentProps
+} from "@patternfly/react-core";
+import { ISortBy } from "@patternfly/react-table";
+import { ConnectionLinksToggleGroup } from "modules/connection-detail/components";
+import { SortForMobileView, useWindowDimensions } from "components";
+
+export interface IConnectionLinksToolbarProps extends DataToolbarContentProps {
+  totalRecords: number;
+  filterSelected?: string;
+  nameSelected?: string;
+  nameInput?: string;
+  addressSelected?: string;
+  addressInput?: string;
+  nameOptions?: any[];
```

Comment:
is it possible to add specific data type besides any[]. any[] used multiple places.

Reply:
no, it is not possible for this case